src/Files.App/UserControls/Pane/InfoPane.xaml

```diff
@@ -238,6 +238,112 @@
 					TextAlignment="Center"
 					TextWrapping="Wrap"
 					Visibility="Collapsed" />
+
+				<!--  Drive Details  -->
+				<StackPanel
+					x:Name="DriveFormatAndType"
+					Margin="12,12,12,24"
+					HorizontalAlignment="Center"
+					Orientation="Horizontal"
+					Spacing="8"
+					Visibility="Collapsed">
+					<TextBlock
+						x:Name="DriveFormatTextBox"
+						HorizontalAlignment="Center"
+						FontFamily="{ThemeResource ContentControlThemeFontFamily}"
+						FontSize="14"
+						IsTextSelectionEnabled="True"
+						Text="{x:Bind ViewModel.SelectedDriveItem.Filesystem, Mode=OneWay}"
+						TextAlignment="Center"
+						TextWrapping="Wrap" />
+					<TextBlock
+						x:Name="DriveTypeTextBox"
+						HorizontalAlignment="Center"
+						FontFamily="{ThemeResource ContentControlThemeFontFamily}"
+						FontSize="14"
+						Foreground="{ThemeResource TextFillColorTertiaryBrush}"
+						IsTextSelectionEnabled="True"
+						Text="{x:Bind ViewModel.SelectedDriveItem.TypeText, Mode=OneWay}"
+						TextAlignment="Center"
+						TextWrapping="Wrap" />
```

Comment:
It looks like there can be some responsiveness issues here and we might need word wrapping. Can these be combined into a single `TextBlock`? You can still achieve the styling by using multiple runs.
![image](https://github.com/user-attachments/assets/006797e5-6b67-4b43-892f-f2036e73b0fb)

Reply:
Is spacing sufficient in the first picture or should I go with the second?

![Screenshot 2024-11-24 172936](https://github.com/user-attachments/assets/87821afc-ea22-4671-b192-f9a13acc8dec)
![Screenshot 2024-11-24 172941](https://github.com/user-attachments/assets/a84eae6f-0fd5-4502-8adc-7e3915e2272c)

Reply:
The issue was less with the spacing and more that it gets cut off on the sides.

Reply:
My bad, I wasn't clear: both the pictures above are using a single `TextBlock`. 
The difference is that in the second one I added a `Run` containing a white space

Reply:
I prefer without the extra run.